portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Malee's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+        #grad1 {            
+            background: linear-gradient(to top right, #ffe5c2 22%, #f1bfc5 75%,  #a58dc1 100% )
+        }
+    </style>
   </head>
-  <body>
+  <div id="grad1">
+  <a href="https://github.com/MSeechan" target="_blank"><img src="images/git.png"/></a>
+  <a href="https://steamcommunity.com/profiles/76561198202975597/" target="_blank"><img src="images/steam.png"/></a> 
+  <body style="font-family: 'Segoe UI'" >
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1><u>Malee's Portfolio</u></h1>
+      <img src="images/selfie.jpg" />
+      <p>Welcome to my portfolio.</p>
+      <p><strong>Hello!</strong> Malee here. It's a pleasure to meet everyone. Here is a little about myself:
+        <br>
+            <ul>
+                <br></br>
```

Comment:
nit: for consistency use `<br>`. Looks like you're already familiar with that above ^

I think`<br / >`  is for XHTML.

Reply:
Can you clarify this nit? Is the issue on line 23 or 25?